src/Suave.Tests/regressions/Bug435.fsx

```diff
@@ -0,0 +1,46 @@
+#r "../bin/Debug/Suave.dll"
```

Comment:
This script runs the server with the sample Owin app so that you can use a browser or some other client to verify that you are getting the Content-Length header. I don't think it needs to be a .fs file since there is already the Bug435-OwinContentLength.fs file, which is the unit test for this issue.